protractor.conf.js

```diff
@@ -29,7 +29,7 @@ const config = {
   capabilities : {
     browserName : 'chrome',
     chromeOptions : {
-      args : ['--disable-gpu', '--window-size=1280,1024'],
+      args : ['--disable-gpu', '--window-size=1280,1024', process.env.CHROME_OPTIONS],
```

Comment:
This is a good idea, however, it doesn't work on my machine.  I tweaked it a bit and this is what works:
```js
const config = { 
  // all the params
};

if (process.env.CHROME_OPTIONS) {
  config.capabilities.chromeOptions.args.push(process.env.CHROME_OPTIONS);
}
```

Reply:
Good catch.  Should have tried it without an environment variable myself!

Reply:
Done